app/src/main/java/org/schabi/newpipe/fragments/list/comments/CommentsFragment.java

```diff
@@ -28,6 +29,7 @@ public class CommentsFragment extends BaseListInfoFragment<CommentsInfoItem, Com
     private final CompositeDisposable disposables = new CompositeDisposable();
 
     private TextView emptyStateDesc;
+    private Button scrollToTopButton;
```

Comment:
The `scrollToTopButton` is not referenced anywhere other than the `initViews` method. You should remove the field and make it a local variable instead.